compiler-rt/lib/asan/tests/CMakeLists.txt

```diff
@@ -206,13 +206,15 @@ function(add_asan_tests arch test_runtime)
           -Wl,-nodefaultlib:libcmt,-defaultlib:msvcrt,-defaultlib:oldnames
         )
     else()
+      set(DYNAMIC_LINK_FLAGS)
```

Comment:
Is this empty set need? I would assume it's only required since the macro doesn't the the output unconditionally? I think it would be better to just always set output_var in get_dynamic_link_flags_for_arch

Reply:
I've handled `DYNAMIC_LINK_FLAGS` like `TARGET_LINK_FLAGS` in this regard.  I guess it would be good to be consistent here.

Reply:
Consistency is important so that is fine by me. Not setting it consistently is a bit weird though so ideally they should both be fixed in a follow-up (or baseline) change.

Reply:
I'd rather go for a baseline change so it's consistent throughout `compiler-rt`.  I may well have a look; however for now I'm busy on the LLVM 19 release ;-)